### PR TITLE
Readme focus on Web Frameworks

### PR DESCRIPTION
**Pull request by johnpapa**
Updating Readme to be more focused on Web than a specific framework.  Fixing typos and grammar a lil too
 #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,69 +28,69 @@ You can customize your project for GitHub Codespaces by committing configuration
...
 ### For the Educator
 
-For you as a teacher that means that you can create an environment, in the cloud, for your class that all students can use with 0 or next to 0 configuration regardless of what operating system they are using.
+Teachers can create an environment, in the cloud, for your class that all students can use with zero or next to zero configuration, regardless of what operating system they are using.
 
-## Codespaces JS template
+## Codespaces JavaScript template
 
-This repo is a GitHub template for a JavaScript frontend web application using the React framework. The goal here is to give you a template to you can immediately utilize for React, or show you hwo you can adapt it for your particular JS needs.
+This repository is a GitHub template for a Web application using the React web framework. The goal here is to give you a template to you can immediately use. You can aslo use the steps in this document to adapt it for your particular needs.
```
Comment (alfredodeza) [inline README.md:42]: Why is `Web` capitalized? 
```suggestion
This repository is a GitHub template for a web application using the React web framework. The goal here is to give you a template to you can immediately use. You can also use the steps in this document to adapt it for your particular needs.
```

**Comment by johnpapa** [comment]
@saragibby  - would be great to apply similar changes to the other javascript repo, too.

**Comment by saragibby** [comment]
Re-framing the readme to "web frameworks" is perfect! Thanks for reading this through and making the changes @johnpapa! I will update the student project to reflect the same.

**Comment by Ivelisse09** [comment]
omg you

**Pull request merged**